cmd/skiff/top.go

```diff
@@ -61,10 +74,65 @@ func (h *FileHeap) Pop() interface{} {
 	return item
 }
 
+// HumanReadableSize converts a byte count to a human readable string
+// From https://yourbasic.org/golang/formatting-byte-size-to-human-readable-format/
+func HumanReadableSize(b int64) string {
+	const unit = 1000
+	if b < unit {
+		return fmt.Sprintf("%d B", b)
+	}
+	div, exp := int64(unit), 0
+	for n := b / unit; n >= unit; n /= unit {
+		div *= unit
+		exp++
+	}
+	return fmt.Sprintf("%.1f %cB",
+		float64(b)/float64(div), "kMGTPE"[exp])
+}
+
+// getFilteredLayers returns only the layers that needs to be
+// processed/extracted e.g. after user specifies specific layer(s)
+// using --layer, we shouldn't be processing all the layers.
+func getFilteredLayers(img types.Image, layers []string) ([]types.BlobInfo, error) {
+	allLayers := img.LayerInfos()
+
+	if len(layers) == 0 {
+		return allLayers, nil // No filtering needed
+	}
```

Comment:
This case needs to be documented as the function signature would suggest that in this case `[], nil` is returned

Reply:
It's there in the docstring, perhaps adding it to -h would be beneficial.